### PR TITLE
Added an accordion to the project

### PR DESCRIPTION
**Pull request by Noibisjunior**
## Changes proposed

I added an accordion to the projects to help visitors understand the projects

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [] My change requires changes to the documentation.
- [] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

link

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2150"></a>

### CODE REVIEW

File: public/data/Noibisjunior.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Abdulsalaam Noibi",
```
Comment (Vyvy-vi) [inline public/data/Noibisjunior.json:2]: it would be better if this profile is added in a separate PR.

File: src/App.js [hunk 2]
```diff
@@ -25,6 +25,16 @@ function App() {
 
   return (
     <Router>
+      <head>
```
Comment (Vyvy-vi) [inline src/App.js:28]: Linkfree is currently using primereact components with tailwindcss for styling. Adding bootstrap would create some redundancy. Perhaps you could implement the same with the Accordion component provided by primereact: https://www.primefaces.org/primereact/accordion/ ?

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
Thanks for opening this PR and showing interest in LinkFree!
I've left a few comments requesting changes. While this is a great feature, adding Bootstrap as an additional dependency might cause issues later on and make the project less maintainable.

We really appreciate that you took out some of your time to work on LinkFree. If you'd like to refactor this and use primereact's Accordian element, feel free to push additional changes to this PR (that would be highly appreciated, if not, then we could refactor this and merge it in).

**Comment by Noibisjunior** [comment]
Thanks for the feedback,i would be extremely grateful if my pull request
can merged,I would try my possible best to refactor the code and  push the
additional changes to the PR.

**Comment by Noibisjunior** [comment]
Please check it out,i have refactor the code base by adding prime react
accordion feature.
https://github.com/Noibisjunior/LinkFree/tree/Accord
please merge

**Comment by EmmaDawsonDev** [comment]
Hi @Noibisjunior thank you so much for your contribution to LinkFree. Unfortunately we just migrated away from React + PrimeReact to NextJS and Tailwind instead.

**Comment by eddiejaoude** [comment]
As we are no longer using PrimeReact, I will close this PR. Thank you for your contribution and sorry it could not be merged this time.

**Pull request closed without merging**